PopupRequest Typescript error in Chapter2 TodoListSPA app

### This issue is for a: (mark with an `x`)
```
- [ x ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> download repo
> cd to repo directory
> cd to Chapter2/TodoListSPA
> npm i
> ng serve 

### Any log messages given by the failure
```
src/app/app.component.ts:58:40 - error TS2345: Argument of type '{ prompt?: string | undefined; authority?: string | undefined; correlationId?: string | undefined; claims?: string | undefined; resourceRequestMethod?: string | undefined; resourceRequestUri?: string | undefined; ... 10 more ...; scopes?: string[] | undefined; } | { ...; }' is not assignable to parameter of type 'RedirectRequest | undefined'.
  Type '{ prompt?: string | undefined; authority?: string | undefined; correlationId?: string | undefined; claims?: string | undefined; resourceRequestMethod?: string | undefined; resourceRequestUri?: string | undefined; ... 10 more ...; scopes?: string[] | undefined; }' is not assignable to type 'RedirectRequest'.
    Type '{ prompt?: string | undefined; authority?: string | undefined; correlationId?: string | undefined; claims?: string | undefined; resourceRequestMethod?: string | undefined; resourceRequestUri?: string | undefined; ... 10 more ...; scopes?: string[] | undefined; }' is not assignable to type '{ scopes: string[]; redirectStartPage?: string | undefined; onRedirectNavigate?: ((url: string) => boolean | void) | undefined; }'.
      Types of property 'scopes' are incompatible.
        Type 'string[] | undefined' is not assignable to type 'string[]'.
          Type 'undefined' is not assignable to type 'string[]'.

58         this.authService.loginRedirect({...this.msalGuardConfig.authRequest});

```

### Expected/desired behavior
>

### Browser and version?
>

### Versions
>

### Mention any other details that might be useful
I'm assuming this is due to an update to the typescript definition for PopupRequest in the msal-browser package. I'm new to this though and am having trouble figuring out what the solution is.

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

--- a/Chapter1/src/app/app.component.ts
+++ b/Chapter1/src/app/app.component.ts
@@ -1,6 +1,6 @@
-import { Component, OnInit, Inject, OnDestroy, Injectable } from '@angular/core';
+import { Component, OnInit, Inject, OnDestroy } from '@angular/core';
 import { MsalService, MsalBroadcastService, MSAL_GUARD_CONFIG, MsalGuardConfiguration } from '@azure/msal-angular';
-import { EventMessage, EventType, InteractionType } from '@azure/msal-browser';
+import { EventMessage, EventType, InteractionType, PopupRequest, RedirectRequest } from '@azure/msal-browser';
 import { Subject } from 'rxjs';
 import { filter, takeUntil } from 'rxjs/operators';
 
@@ -47,7 +47,7 @@ export class AppComponent implements OnInit, OnDestroy {
   login() {
     if (this.msalGuardConfig.interactionType === InteractionType.Popup) {
       if (this.msalGuardConfig.authRequest){
-        this.authService.loginPopup({...this.msalGuardConfig.authRequest})
+        this.authService.loginPopup({...this.msalGuardConfig.authRequest} as PopupRequest)
           .subscribe(() => this.checkAccount());
         } else {
           this.authService.loginPopup()
@@ -55,7 +55,7 @@ export class AppComponent implements OnInit, OnDestroy {
       }
     } else {
       if (this.msalGuardConfig.authRequest){
-        this.authService.loginRedirect({...this.msalGuardConfig.authRequest});
+        this.authService.loginRedirect({...this.msalGuardConfig.authRequest} as RedirectRequest);
       } else {
         this.authService.loginRedirect();
       }

--- a/Chapter2/TodoListSPA/src/app/app.component.ts
+++ b/Chapter2/TodoListSPA/src/app/app.component.ts
@@ -1,6 +1,6 @@
 import { Component, OnInit, Inject, OnDestroy, Injectable } from '@angular/core';
 import { MsalService, MsalBroadcastService, MSAL_GUARD_CONFIG, MsalGuardConfiguration } from '@azure/msal-angular';
-import { EventMessage, EventType, InteractionType } from '@azure/msal-browser';
+import { EventMessage, EventType, InteractionType, PopupRequest, RedirectRequest } from '@azure/msal-browser';
 import { Subject } from 'rxjs';
 import { filter, takeUntil } from 'rxjs/operators';
 
@@ -47,7 +47,7 @@ export class AppComponent implements OnInit, OnDestroy {
   login() {
     if (this.msalGuardConfig.interactionType === InteractionType.Popup) {
       if (this.msalGuardConfig.authRequest){
-        this.authService.loginPopup({...this.msalGuardConfig.authRequest})
+        this.authService.loginPopup({...this.msalGuardConfig.authRequest} as PopupRequest)
           .subscribe(() => this.checkAccount());
         } else {
           this.authService.loginPopup()
@@ -55,7 +55,7 @@ export class AppComponent implements OnInit, OnDestroy {
       }
     } else {
       if (this.msalGuardConfig.authRequest){
-        this.authService.loginRedirect({...this.msalGuardConfig.authRequest});
+        this.authService.loginRedirect({...this.msalGuardConfig.authRequest} as RedirectRequest);
       } else {
         this.authService.loginRedirect();
       }

